.ci_support/migrations/hdf51140.yaml

```diff
@@ -1,7 +0,0 @@
-__migrator:
-  build_number: 1
-  kind: version
-  migration_number: 1
-hdf5:
-- 1.14.0
-migrator_ts: 1675622758
```

Comment:
The `hdf5` migrator was updated recently ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4594 ), which has caused it to be removed in some cases ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4594#issuecomment-1609308017 )